data/categories/vp_vehicles_parts.yml

```diff
@@ -4453,6 +4453,160 @@
   - upholstery_pattern
 - id: vp-2-1
   name: Aircraft
+  children:
+  - vp-2-1-1
+  - vp-2-1-2
+  - vp-2-1-3
+  - vp-2-1-4
+  attributes:
+  - body_color
+  - body_pattern
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-1-1
+  name: Commercial Aircraft
+  children:
+  - vp-2-1-1-1
+  - vp-2-1-1-2
+  attributes:
+  - body_color
+  - body_pattern
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-1-1-1
+  name: Passenger Jets
+  children: []
+  attributes:
+  - body_color
+  - body_pattern
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-1-1-2
+  name: Cargo Aircraft
+  children: []
+  attributes:
+  - body_color
+  - body_pattern
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-1-2
+  name: Private Jets
+  children:
+  - vp-2-1-2-1
+  - vp-2-1-2-2
```

Comment:
I don't think it makes sense for `Turboprop Aircraft` to be a child of `Private Jets`:
* Turboprops aren't jets by definition (different engine type)
* A turboprop can also fall under `Commercial Aircraft` (e.g. a Q400 passenger plane)